src/ray/util/test_util.h

```diff
@@ -37,6 +38,14 @@ inline TaskID RandomTaskId() {
   return TaskID::FromBinary(data);
 }
 
+/// Utility function to convert a buffer to a string. This makes a copy
+/// and should only be used for testing.
+/// \param[in] buffer The buffer to convert.
+/// \return A string with a copy of the buffer content.
+std::string BufferToString(const Buffer& buffer) {
```

Comment:
Instead of this, do you think it's better to add a `GetMetadataAsString` in `RayObject`? because metadata is usually short, and we may want to read it as a string in non-testing code as well.